docs/contribute/i18n.rst

```diff
@@ -17,3 +17,50 @@ https://www.transifex.com/organization/elgg
 Plugin authors are encouraged to coordinate translations via Transifex as well
 so the whole community can be unified and make it really easy for translators
 to contribute to any plugin in the Elgg ecosystem.
+
+Pulling translations
```

Comment:
can you also add instructions for pushing? maybe suggest pushing also before pulling, so you will have the latest updates

Reply:
Can both be done via a composer script? We could then configure either travis or scrutinizer to create a pull request after a merge.

Reply:
Pushing should happen only when creating a completely new Transifex project. Do we have any instructions about starting a new major Elgg version? That would be the best place for docs about pushing.

Yes, pulling translations could be automated. But every time after a merge sounds a bit too often?

Reply:
> Pushing should happen only when creating a completely new Transifex project

why? minor updates or even bugfixes could add new translations am i right? So a push could happen any time

Reply:
Ah, you mean pushing new translation **keys** into Transifex? I though you mean pushing the actual translations. Transifex is able to fetch new keys automatically from Github.

This has been done for the Elgg 1 & 2 Transifex projects but not yet for the Elgg 3 project. It can be done once we branch `master` into a dedicated Elgg 3 branch.

Reply:
> Transifex is able to fetch new keys automatically from Github.

if this is the case, could you document/mention somewhere how/that this works

Reply:
Instructions added.